html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Forms</title>
+</head>
+<body>
+    <h1>Coding Form!</h1>
+        <form>
+            <label for="email">Email</label>
+                <div>
+                    <input type="email" id="email">
+                </div>
+        </form>
+
+        <form>
```

Comment:
You don't want every input to be in its own `form` element like this, all of the inputs are for one form, so they should all be wrapped by only one `form` element.

Reply:
I understand.. resolved